src/components/FAQ/FAQ.js

```diff
@@ -1,61 +1,60 @@
 import React from 'react';
-import './FAQ.scss';
+import './FAQ.css';
```

Comment:
The styling isn't working correctly because this file name is incorrect. Change it back to `*.scss` and it should work better.

Reply:
Make sure to compile your code too, because that's where the error's caught again.